src/controllers/v1/match2/index.ts

```diff
@@ -345,6 +345,25 @@ export class Match2Controller extends Controller {
     return transaction
   }
 
+  /**
+   * @summary Get all of a match2's cancellation transactions
+   * @param match2Id The match2's identifier
```

Comment:
you can also specify a type in curly brackets e.g. `{String}` which would get highlighted, since it's UUID, maybe we would like to stick to the UUID format. In that case it would be nice to have a tests as well for valid input

Reply:
doe this relate to the comment on 349-350?